test/model/calculations.test.ts

```diff
@@ -10,56 +10,78 @@ describe(Calculations.OutstandingStockSharesCalculator, () => {
 
   test("issuances", () => {
     const subject = new Calculations.OutstandingStockSharesCalculator();
-    subject.apply(fakeStockTxn("ISSUANCE", { quantity: "10" }));
-    subject.apply(fakeStockTxn("ISSUANCE", { quantity: "1" }));
+    subject.apply(fakeStockTxn("STOCK_ISSUANCE", { quantity: "10" }));
+    subject.apply(fakeStockTxn("STOCK_ISSUANCE", { quantity: "1" }));
     expect(subject.value).toBe(11);
   });
 
+  test("plan issuances", () => {
```

Comment:
**Please fix:** I see a lack of tests for other `TX_PLAN_SECURITY_...` transactions besides ISSUANCE and RETRACTION. Even if the logic is identical, we should have a test for every txn type.

Reply:
I planned to do that in my next PR. But I can do add them now